### PR TITLE
Fix/agreement generator tests

### PR DESCRIPTION
**Pull request by rosogon**
This PR adds the missing tests for AgreementGenerator in #295.

### CODE REVIEW

**Comment by rosogon** [comment]
@PaoloCifariello , @kiuby88 : Could you have a look?

File: planner/core/src/test/java/eu/seaclouds/platform/planner/core/application/agreements/AgreementGeneratorTest.java [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+    @Test
+    public void testGenerateAgreementId() throws Exception {
+        /*
+         * Start the mock server
+         */
+        MockWebServer server = new MockWebServer();
+        try {
+
+            server.enqueue(new MockResponse().setBody(
+                    readFile("/eu/seaclouds/planner/core/application/agreements/sla-create-template-response.json")));
```
Comment (kiuby88) [inline planner/core/src/test/java/eu/seaclouds/platform/planner/core/application/agreements/AgreementGeneratorTest.java]: Minor: could this ugly path be avoided using resources object?

File: planner/core/src/test/java/eu/seaclouds/platform/planner/core/application/agreements/AgreementGeneratorTest.java [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+    @Test
+    public void testGenerateGetAgreement() throws Exception {
+        /*
+         * Start the mock server
+         */
+        MockWebServer server = new MockWebServer();
+        try {
+
+            server.enqueue(new MockResponse().
+                    setBody(readFile("/eu/seaclouds/planner/core/application/agreements/sla-create-agreement-response.xml")));
```
Comment (kiuby88) [inline planner/core/src/test/java/eu/seaclouds/platform/planner/core/application/agreements/AgreementGeneratorTest.java]: Same above

**Comment by kiuby88** [comment]
Thanks a lot @rosogon for completing #295 and adding some test, even though it was not your responsability...

**Pull request merged**